app/src/main/java/de/xikolo/views/ExoPlayerVideoView.kt

```diff
@@ -166,8 +193,7 @@ open class ExoPlayerVideoView : PlayerView {
         exoplayer.setPlaybackParameters(
             PlaybackParameters(
                 speed,
-                exoplayer.playbackParameters.pitch,
-                exoplayer.playbackParameters.skipSilence
```

Comment:
The `skipSilence` is not required anymore? I know it is not part of the constructor anymore, but does it need to be set somehow else?

Reply:
SkipSilence was never required to be set, that's why the new PlaybackParams with the updated speed includes the old value, just like for the pitch.